dali/python/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 # Get all the srcs
+collect_headers(DALI_INST_HDRS PARENT_SCOPE)
 file(GLOB tmp *.cc)
```

Comment:
why not use collect sources for those `*.cc`?

Reply:
Done